symbols.xml

```diff
@@ -219,22 +219,22 @@
         generated standingorders
 
         <global-address name='standing_orders_mix_food' value='0x00f9303d'/>
-        <global-address name='standing_orders_gather_bodies' value='0x00f9303e'/>
```

Comment:
Yeah, they're definitely not all the same address. :)

You should still run `devel/find-offsets standing_orders`, though, just to verify that they're correct.